tests/buyers/helpers/test_buyers_helpers.py

```diff
@@ -101,6 +101,28 @@ def test_is_brief_correct(self):
             'digital-specialists',
             123
         ) is False
+        assert helpers.buyers_helpers.is_brief_correct(
+            api_stubs.brief(user_id=123, status='withdrawn')['briefs'],
+            'digital-outcomes-and-specialists',
+            'digital-specialists',
+            123,
+            allow_withdrawn=True
+        ) is True
+
+    @pytest.mark.parametrize(
+        ['status', 'allow_withdrawn', 'result'],
+        [
+            ('withdrawn', True, True),
```

Comment:
This first test scenario is covered by the one you have just written above.
The second test scenario is the same logically but is testing the default argument.
Would be nice to have all the tests regarding status of the briefs be done in the same way (i.e. moving the above two tests into the parametrized version).